cmd/docs.go

```diff
@@ -37,18 +39,33 @@ var docsCmd = &cobra.Command{
 				u.LogErrorAndExit(schema.CliConfiguration{}, err)
 			}
 
+			// Detect terminal width if width flag is not set
+			width := cliConfig.Settings.Docs.MaxWidth
+			if width == 0 {
+				if term.IsTerminal(int(os.Stdout.Fd())) {
+					w, _, err := term.GetSize(int(os.Stdout.Fd()))
+					if err == nil {
+						width = int(w)
+					}
+
+					if width > 120 {
+						width = 120
+					}
```

Comment:
Seems like we're not using max width?
```suggestion
					if w > width {
						w = width
					}
```

Reply:
Updated the naming!